mis_builder/static/src/components/mis_report_widget.esm.js

```diff
@@ -104,11 +105,16 @@ export class MisReportWidget extends Component {
     }
 
     get context() {
-        var ctx = super.context;
-        if (this.showSearchBar) {
+        let ctx = this.props.record.context;
```

Comment:
@IT-Ideas why is it not a problem to not call super here?

Reply:
Yes it would for sure. But from what I know, Component doesn't have a context property. Am I wrong?

Reply:
> Component doesn't have a context property

Indeed :)